atlasdb-client/src/main/java/com/palantir/atlasdb/util/MetricsManager.java

```diff
@@ -65,6 +70,14 @@ public void registerMetric(Class clazz, String metricName, Metric metric) {
         registerMetricWithFqn(MetricRegistry.name(clazz, metricName), metric);
     }
 
+    public void registerTaggedGauge(Class clazz, String metricName, Map<String, String> tag, Gauge gauge) {
```

Comment:
If we move the `Map<String, String> tag` to last argument and change the method name to just `registerMetric` then we can just go around adding tags to the end of anywhere we currently register metrics and so minimise the effort to add tags.